match2/commons/match_group_display_bracket.lua

```diff
@@ -130,7 +129,7 @@ function BracketDisplay.Bracket(props)
 			:node(BracketDisplay.NodeBody(nodeProps))
 	end
 
-	return html.create('div'):addClass('brkts-main brkts-main-dev')
+	return html.create('div'):addClass('brkts-main brkts-main-dev brkts-main-dev-2')
```

Comment:
I've mentioned this in an earlier PR this is based on, but this seems a bit weird